docs/source/tutorials/e2e_flow.rst

```diff
@@ -348,101 +411,143 @@ conversion is that you can directly work with standard formats. This helps
 with interoperability with other libraries since torchtune doesn't add yet
 another format to the mix.
 
-Let's take a look at an example of how this would work with a popular codebase
-used for running performant inference with LLMs -
-`gpt-fast <https://github.com/pytorch-labs/gpt-fast/tree/main>`_. This section
-assumes that you've cloned that repository on your machine.
+Let's start with huggingface
```

Comment:
nit: be consistent with naming. It should be "Hugging Face" everywhere in this section, not huggingface or HF.